src/lib.rs

```diff
@@ -6,7 +6,7 @@
 
 include!("bindings.rs");
 
-#[cfg(feature = "static")]
+#[cfg(feature = "vendor-libbpf")]
```

Comment:
Looks like there's some mismatch around feature naming: `vendor-blah` vs `vendored-blah`.

Reply:
Oh yes, that's a good catch. Thanks, fixed.